buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -672,14 +667,6 @@ class BuildPlugin implements Plugin<Project>  {
     }
 
     private static void configurePom(Project project, MavenPublication publication) {
-        // Set the pom's destination to the distribution directory
-        project.tasks.withType(GenerateMavenPom).all { GenerateMavenPom pom ->
-            if (pom.name == "generatePomFileFor${publication.name.capitalize()}Publication") {
-                BasePluginExtension baseExtension = project.getExtensions().getByType(BasePluginExtension.class);
-                pom.destination = project.provider({"${project.buildDir}/distributions/${baseExtension.archivesName.get()}-${project.getVersion()}.pom"})
```

Comment:
this results in a deprecation warning and we would not want to rely on that workaround anymore.